### DISCUSSION TITLE
How to make <Head> work with `v-html`?

### DISCUSSION THREAD

**Discussion by a1vin1au** [inertiajs/inertia#1867] [Help (Vue)]
We want to add HTML editor in our admin for set any HTML, e.g. `<meta>`, `preload` etc... then we also enabled SSR on server, how can I put HTML codes into `<Head>`?

```vue
<script setup>
import { Head } from '@inertiajs/vue3'
// ...

const html = ref('<meta name="hello" content="world" />');
</script>

<template>
  <Head v-html="html"></Head>
</template>
```

**Comment by a1vin1au** [comment]
Hello, anyone can help? 🥹🥹

**Comment by darthsoup** [comment]
This does not work, Head is only intended for managing the page title.
Try to achieve this with https://github.com/unjs/unhead
